<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#usuario_evaluado{
  border-width: 1px;
  border-style: solid;
  border-color: black;
}

#proceso_evaluado{
  border-width: 1px;
  border-style: solid;
  border-color: black;
}

#tipo_evaluacion{
  border-width: 1px;
  border-style: solid;
  border-color: black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../media/imagenes/logo.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="usuario_evaluado">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="proceso_evaluado">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tipo_evaluacion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="space_preguntas">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>981</width>
      <height>421</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_evaluation">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>660</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_evaluation">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>660</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_evaluation">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>660</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
